src/tests/ftest/dfuse/daos_build.py

```diff
@@ -35,12 +20,36 @@ class DaosBuild(DfuseTestBase):
     :avocado: recursive
     """
 
-    @skip_on_centos7()
-    def test_daos_build(self):
-        """Jira ID: DAOS-8937.
+    def test_writeback(self):
+        """ This test builds DAOS on a dfuse filesystem.
+        Use cases:
+            Create Pool
+            Create Posix container
+            Mount dfuse
+            Checkout and build DAOS sources.
+        :avocado: tags=all,pr
+        :avocado: tags=hw,small
+        :avocado: tags=daosio,dfuse
+        :avocado: tags=dfusedaosbuild,dfusedaosbuild_wb
```

Comment:
with the "test_" prefix on the tag?

Reply:
Yes, specifically with the test_ prefix. The idea is that if all tests had a unique tag that was _**exactly the same**_ as the method name, it would be much easier to figure out how to run a single test